src/components/shared/Button.tsx

```diff
@@ -12,28 +12,53 @@ import styled from 'styled-components/native';
 
 import { ratio, colors } from '../../utils/Styles';
 
+interface IStyledElement {
+  white?: boolean;
+};
+
+interface IButtonContainer {
+  width: number;
+  height: number;
+};
+
+interface IProps {
+  testID: string;
+  containerStyle: ViewStyle;
+  isWhite: boolean;
+  isLoading: boolean;
+  isDisabled: boolean;
+  onPress: () => void;
+  imgLeftSrc: any;
+  indicatorColor: string;
+  activeOpacity: number;
+  children: any;
+  width: number;
+  height: number;
+}
+
 const StyledContainer = styled.View`
   flex: 1;
 `;
 
-const StyledButton = styled.View`
-  background-color: ${colors.dodgerBlue};
+const StyledButtonContainer = styled.View<IButtonContainer>`
```

Comment:
I'm sorry but could you tell `<IButtonContainer>` is for here? I feel like this is quite useless.